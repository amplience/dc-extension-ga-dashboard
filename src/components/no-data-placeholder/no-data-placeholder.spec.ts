import { render } from '@testing-library/svelte';
import NoDataPlaceholder from './no-data-placeholder.svelte';

describe('Loader', () => {
  it('should render the Loader component', () => {
    const { container } = render(NoDataPlaceholder, {});

    expect(container.firstChild).toMatchSnapshot();
  });
});
