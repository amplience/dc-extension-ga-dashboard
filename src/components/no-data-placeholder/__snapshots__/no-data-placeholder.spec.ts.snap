// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render the Loader component 1`] = `
<div>
  <div
    class="no-data-container svelte-1kq81j9"
  >
    <p>
      No data for the selected date range
    </p>
  </div>
</div>
`;
